charts/photoprism/values.yaml

```diff
@@ -0,0 +1,62 @@
+# Default values for PhotoPrism.
+
+image:
+  repository: photoprism/photoprism
+  pullPolicy: IfNotPresent
+  tag: "20201122"
+
+strategy:
+  type: Recreate
+
+service:
+  port:
+    port: 2342
+
+env: {}
+  # PHOTOPRISM_ADMIN_PASSWORD: "please-change"     # Initial admin password: PLEASE CHANGE!
```

Comment:
No need to include all the envars, only the essentials like https://github.com/k8s-at-home/charts/blob/master/charts/traccar/values.yaml#L15

You can link to https://raw.githubusercontent.com/photoprism/photoprism/master/docker/photoprism/docker-compose.yml

Reply:
resolved by df1a3f4

Reply:
Also added a link to PhotoPrism config options directly in values.yaml